config/application_template.properties

```diff
@@ -24,7 +24,7 @@ spring.datasource.validationQuery=SELECT 1
 #spring.datasource.test-on-borrow=true
 #spring.datasource.validationQuery=SELECT 1 from dual
 
-ctools.direct.url=
+ctools.server.url=
```

Comment:
Is this the name of the server or the load balancer DNS name.  Just want to confirm that it is the actual server and isn't taken from the configuration file.